team/bundles/org.eclipse.compare.win32/src/org/eclipse/compare/internal/win32/WordComparison.java

```diff
@@ -304,6 +316,19 @@ private OleAutomation getActiveDocument(OleAutomation application) {
 		return getAutomationProperty(application, "ActiveDocument"); //$NON-NLS-1$
 	}
 
+	private int getWordVersion(OleAutomation application) {
```

Comment:
Not sure if it is worth it adding some error handling for the case that parsing of Word (Office) version would fail here. What do you think?